doc/nvim-tree-lua.txt

```diff
@@ -192,11 +192,13 @@ Subsequent calls to setup will replace the previous configuration.
           },
         },
         float = {
+          -- enabling enforces `actions.open_file.quit_on_open`
           enable = false,
           -- options passed to nvim_open_win()
           relative = "editor",
           row = 1,
           col = 1,
+          anchor = "NW",
           border = "rounded",
         },
```

Comment:
please document these option. See `:help nvim-tree.view` force instance to see how its done :)

Reply:
Done.